__tests__/app.test.js

```diff
@@ -0,0 +1,49 @@
+const request = require("supertest")
+const app = require('../app.js');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const testData = require('../db/data/test-data')
+
+beforeEach(() => {
+    return seed(testData);
+})
+
+afterAll(() => {
+    db.end();
+});
+
+
+describe('app - working', () => {
+    describe('GET /api/topics', () => {
+        test('Should return json of the topics with status 200', () => {
+            return request(app)
+            .get('/api/topics')
+            .expect(200)
+            .then(({body}) => {
+                topics = body.topics;
+                expect(topics).toBeInstanceOf(Array);
+                topics.forEach((topic) => {
+                    expect(topic).toEqual(
+                        expect.objectContaining({
+                            slug: expect.any(String),
+                            description: expect.any(String)
+                        })
+                    );
```

Comment:
Id like to see some testing for the length of the array. A forEach testing an empty array will always pass so we should make sure that the array actually has things in it as well! :)